WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -48,19 +48,15 @@
     <% if competitions.empty? %>
       <%= render 'no_competitions_found' %>
     <% else %>
-      <% past_competitions, ongoing_and_upcoming_competitions = competitions.partition(&:is_over?) %>
-      <% recent_competitions, distant_competitions = past_competitions.partition { |c| (Date.today - c.end_date).to_i < Competition::RECENT_DAYS } %>
-      <% unless ongoing_and_upcoming_competitions.empty? %>
-        <div class="col-md-12" id="upcoming-comps">
-          <%= render 'admin_index_table', competitions: ongoing_and_upcoming_competitions, title: "Ongoing and upcoming competitions" %>
-        </div>
+      <% if params[:state] == 'past' %>
```

Comment:
Haha, you've fixed what I meant (`params[:state] == 'past'` call for competitions list part) but you left the one I commented.